packages/zendesk-adapter/src/config.ts

```diff
@@ -840,6 +847,8 @@ export const DEFAULT_TYPES: ZendeskApiConfig['types'] = {
         { fieldName: 'id', fieldType: 'number' },
       ],
       fieldsToHide: FIELDS_TO_HIDE.concat({ fieldName: 'id', fieldType: 'number' }),
+      fieldsToOmit: FIELDS_TO_OMIT.concat({ fieldName: 'title', fieldType: 'string' },
+        { fieldName: 'description', fieldType: 'string' }),
```

Comment:
styling nit (here and in other places that have more than one item):
```suggestion
      fieldsToOmit: FIELDS_TO_OMIT.concat(
        { fieldName: 'title', fieldType: 'string' },
        { fieldName: 'description', fieldType: 'string' },
      ),
```